avocado/core/job.py

```diff
@@ -311,7 +311,8 @@ def _update_latest_link(self):
         """
         def soft_abort(msg):
             """ Only log the problem """
-            LOG_JOB.warning("Unable to update the latest link: %s" % msg)
```

Comment:
The intended way to use all logging functions is to pass the format string, and then arguments.  That is, this should be:

```
LOG_JOB.warning("Unable to update the latest link: %s, msg)
```

Or

```
LOG_JOB.warning("Unable to update the latest link: {}, msg)
```